packages/browser/src/sdk.ts

```diff
@@ -166,3 +174,67 @@ export function close(timeout?: number): PromiseLike<boolean> {
 export function wrap(fn: (...args: any) => any): any {
   return internalWrap(fn)();
 }
+
+/**
+ * Enable automatic Session Tracking for the initial page load.
+ */
+function startSessionTracking(): void {
+  const window = getGlobalObject<Window>();
+  const hub = getCurrentHub();
+
+  /**
+   * We should be using `Promise.all([windowLoaded, firstContentfulPaint])` here,
+   * but, as always, it's not available in the IE10-11. Thanks IE.
+   */
+  let loadResolved = document.readyState === 'complete';
+  let fcpResolved = false;
+  const possiblyEndSession = (): void => {
+    if (fcpResolved && loadResolved) {
+      hub.endSession();
+    }
+  };
+  const resolveWindowLoaded = (): void => {
+    loadResolved = true;
+    possiblyEndSession();
+    window.removeEventListener('load', resolveWindowLoaded);
+  };
+
+  hub.startSession();
+
+  if (!loadResolved) {
+    // IE doesn't support `{ once: true }` for event listeners, so we have to manually
+    // attach and then detach it once completed.
+    window.addEventListener('load', resolveWindowLoaded);
+  }
+
+  try {
+    const po = new PerformanceObserver((entryList, po) => {
+      entryList.getEntries().forEach(entry => {
+        if (entry.name === 'first-contentful-paint' && entry.startTime < firstHiddenTime) {
```

Comment:
```suggestion
        if (entry.name === 'first-contentful-paint' && entry.startTime < firstHiddenTime && !fcpResolved) {
```

Reply:
Should guaranteed that the if condition is run at most once.

Reply:
Observer is disconnected after the first call, so I'm not sure how it'd be ever called twice?